.github/workflows/README.md

```diff
@@ -14,6 +14,16 @@ workloads run using [GitHub self-hosted runners](https://help.github.com/en/acti
 5. Run the following command to install dependencies
     - `wget -O - https://raw.githubusercontent.com/GoogleCloudPlatform/stackdriver-sandbox/master/.github/workflows/install-dependencies.sh | bash`
 
+## Setup - Push Enabled
+
+Some actions require authentication to GCP to push container images. These workers need extra setup
+
+**Note:** only one worker in the pool requires this additional setup
```

Comment:
Because the worker that runs these jobs needs more security clearance than the others. The image pushing jobs are rare, so we won't ever have a backlog. It seems like a good security practice to only give the access to the one that needs it

I updated the comment to explain this better too